pkg/ingester/ingester.go

```diff
@@ -1689,6 +1799,8 @@ func (i *Ingester) closeAllTSDB() {
 		go func(db *userTSDB) {
 			defer wg.Done()
 
+			ephemeral := db.getEphemeralStorage() != nil
```

Comment:
this same line exists in two places, maybe there could be a `db.hasEphemeralStorage()` method returning a `bool` directly?